pyro/primitives.py

```diff
@@ -424,12 +424,27 @@ def _fn():
 
 
 def enable_validation(is_validate=True):
+    """
+    Enable or disable validation checks in Pyro. Validation checks provide
+    useful warnings and errors, e.g. NaN checks, validating distribution
+    arguments and support values, etc. which is useful for debugging.
+    Since some of these checks may be expensive, we recommend turning
+    this off for mature models.
+
+    :param bool is_validate: enable validation checks if True.
+    """
     dist.enable_validation(is_validate)
     infer.enable_validation(is_validate)
 
 
 @contextmanager
 def validation_enabled(is_validate=True):
+    """
+    Context manager that is useful when temporarily enabling/disabling
+    validation checks.
+
+    :param bool is_validate: temporary validation check override.
```

Comment:
nit: itd be nice to include a "defaults to True", to denote it is an optional kwarg